app/lib/actions/scc_manager/subscribe_product.rb

```diff
@@ -37,12 +38,15 @@ def humanized_name
     end
 
     class CreateProduct < Actions::Base
+      SCC_GPG_KEY_NAME = 'SUSE Subscriptions Manager GPG Key'.freeze
```

Comment:
This constant might need to be stored at another global place, such that all dependencies can be created automatically.

Reply:
If the current approach is that the admin adds the GPG-Key ContentCredential manually, it might be easier to provide a drop-down select-field to select one of the Content Credentials directly instead of using a requiring to find one with a certain name.

Reply:
Done. Please see description above.

Reply:
Looks good, thanks